backend/inspirehep/records/api/authors.py

```diff
@@ -70,7 +71,8 @@ def get_linked_author_records_uuids_if_author_changed_name(self):
     @classmethod
     def create(cls, data, id_=None, *args, **kwargs):
         record = super().create(data, id_, **kwargs)
-        record.assign_author_to_papers()
+        if current_app.config.get("FEATURE_FLAG_ENABLE_ASSIGN_AUTHOR_PAPERS", False):
```

Comment:
no need for the default `False` here, by default it will return none which is false